docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md

```diff
@@ -169,12 +169,12 @@ $ source cmake-install/bin/enable-cmake.sh
 $ cmake --version
 cmake version 3.15.5
 ```
-Now you have an applicable CMake ready to use. At any time, you could run `source cmake-install/bin/disable-cmake.sh` to disable it.
+Now you have an applicable CMake ready to use. At any time, you could run the command `source cmake-install/bin/disable-cmake.sh` to disable it.
 
 ## Install an Applicable GCC
 For users who don't have an usable GCC installation, we provide a prebuilt GCC and a script to automatically download and install it. Assuming you are now at the build directory, run:
```

Comment:
a usable

Reply:
Well , I did some searching, you are right.